go.mod

```diff
@@ -7,16 +7,16 @@ require (
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/onsi/ginkgo v1.11.0
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.18.0-beta.2
-	k8s.io/apiextensions-apiserver v0.18.0-beta.2
-	k8s.io/apimachinery v0.18.0-beta.2
-	k8s.io/client-go v0.18.0-beta.2
-	k8s.io/code-generator v0.18.0-beta.2
+	k8s.io/api v0.19.0
+	k8s.io/apiextensions-apiserver v0.19.0
+	k8s.io/apimachinery v0.19.0
+	k8s.io/client-go v0.19.0
+	k8s.io/code-generator v0.19.0
 	k8s.io/klog v1.0.0
```

Comment:
Should we migrate to klog/v2 to be in sync with kube 1.19.x ?

Reply:
Good point. We shouldn't mix klog/v1 and v2.

I'd suggest creating one PR to upgrade to klog/v2, and then another one to upgrade to k8s 1.19.